dapps/marketplace/src/utils/distanceToNow.js

```diff
@@ -1,9 +1,13 @@
-export default function distanceToNow(timestamp, long) {
+import { fbt } from 'fbt-runtime'
+
+export default function distanceToNow(timestamp, { long, showJustNow } = {}) {
```

Comment:
There are a bunch of other places in the source that use this function... do we need to update them too?

Reply:
I've done [a search](https://github.com/OriginProtocol/origin/search?q=distanceToNow&unscoped_q=distanceToNow) and none of them pass the 2nd argument.